addons/mail/wizard/mail_activity_schedule_views.xml

```diff
@@ -69,7 +69,7 @@
                     </footer>
                     <footer invisible="not plan_id">
                         <button name="action_schedule_plan" string="Schedule" type="object" class="btn-primary"
-                                invisible="has_error" data-hotkey="q"/>
```

Comment:
** for testing purposes this is not going to merge **

Reply:
This can be done in the templates, in Technical settings if needed :) You can remove it at this stage